chunkenc/chunk.go

```diff
@@ -58,9 +58,16 @@ type Appender interface {
 
 // Iterator is a simple iterator that can only get the next value.
```

Comment:
This is no longer true

Reply:
yup

Reply:
Wonder if we can totally remove `SeriesIterator` everywhere then :thinking:

Reply:
and we have even third SeriesIterator in `tsdbutil`!